src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,106 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Manufacturer volvo = new Manufacturer(1L, "volvo", "Sweden");
+        Manufacturer bmw = new Manufacturer(2L, "bmw", "germany");
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        System.out.println(manufacturerService.create(volvo));
+        System.out.println(manufacturerService.create(bmw));
+
+        Driver bob = new Driver();
+        bob.setLicenseNumber("12345gfhhghgf");
+        bob.setName("Bob");
+        bob.setId(11L);
+        Driver roma = new Driver();
+        roma.setLicenseNumber("gagagsdgsadgasdg");
+        roma.setName("Roma");
+        roma.setId(14L);
+        Driver alice = new Driver();
+        alice.setName("Alice");
+        alice.setLicenseNumber("6789gfhgfd");
+        alice.setId(12L);
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        driverService.create(bob);
+        driverService.create(alice);
+        driverService.create(roma);
+
+        List<Driver> drivers1 = new ArrayList<>();
```

Comment:
Don't use numbers in values name
```suggestion
        List<Driver> drivers = new ArrayList<>();
```